packages/server/database/migrations/20200828161500-samlDomainAsId.ts

```diff
@@ -0,0 +1,26 @@
+import {R} from 'rethinkdb-ts'
+export const up = async function(r: R) {
+  try {
+    await r
+      .table('SAML')
+      .update((row) => ({
+        id: row('domain')
+      }))
+      .run()
+  } catch (e) {
+    console.log(e)
+  }
+  try {
+    await r
+      .table('SAML')
+      .filter((row) => row('id').ne(row('domain')))
+      .delete()
+      .run()
+  } catch (e) {
+    console.log(e)
+  }
+}
+
+export const down = async function() {
+  // noop
```

Comment:
to migrate down, just clone the rows but don't set an ID so rethinkdb generates an ID on its own